openapi-tokenizer.yaml

```diff
@@ -0,0 +1,127 @@
+openapi: 3.0.1

```

Comment:
these specs must be taken from the original URL (tokenizer exposed json)

you may want to convert the json to yaml using https://www.npmjs.com/package/json2yaml